services/utils.py

```diff
@@ -66,6 +66,11 @@ def absolute_url(path):
     return '%s://%s%s' % (scheme, site.domain, path)
 
 
+def get_path_to_service(service_id):
+    """In lieu of a url-reversing mechanism for routes in the app"""
+    return '/app/index.html#/service/%d' % service_id
```

Comment:
Not working locally, for reasons I don't understand.

http://localhost:4005/app/#/service/195 => 404, "Directory indexes are not allowed here."

because the browser makes this request:

```
[29/Jan/2016 22:30:38] "GET /app/ HTTP/1.1" 404 1608
```